service/src/test/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseServiceTest.java

```diff
@@ -442,4 +444,47 @@ void shouldRemoveHearingBookingWithSingleHearing() {
 
         }
     }
+
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Nested
+    class RemoveGatekeepingOrderUrgentHearingOrder {
+
+        private final String migrationId = "DFPL-1162";
+        private final long caseId = 1673628190034209L;
```

Comment:
looks like they can be removed.

Reply:
U mean admin can remove the urgent hearing order by the tools? The regular one can be removed by the removal tools, but user confirmed that they can't remove the order.